insights/parsers/freeipa_healthcheck_log.py

```diff
@@ -0,0 +1,23 @@
+"""
+FREEIPA HealthCheck log
+================================================
+
+"""
+
+from .. import parser
+from insights.specs import Specs
+from insights import JSONParser
+
+
+@parser(Specs.freeipa_healthcheck_log)
+class FreeIPAHealthCheckLog(JSONParser):
+    """Class to parse /var/log/ipa/healthcheck/healthcheck.log"""
```

Comment:
@fcami please follow the [documentation guidelines for the project](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) to ensure that your parser documentation is accurate and renders correctly.

Please include testing for your examples you add to the documentation.

Reply:
@bfahr I've updated the PR, however as you know healthcheck outputs a very long list of long dictionaries that will make either reading or testing the actual docstring near to impossible.
I've included a complete example in the parser though and I'm testing the string in the test.
Let me know if you think any changes should be made.

Reply:
@fcami please refer to the documentation guidelines above about including a `docs/shared_parsers/freeipa_healthcheck_log.rst` file so that your parser will be included in the documentation build.  Please make sure to check that it builds and looks correct.

Reply:
hi @bfahr thanks for the heads up, I've added the file and it renders ok.